recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/displaymanager/displaymanager_git.bbappend

```diff
@@ -1,4 +1,4 @@
-SRCREV = "6cbf0acec5d33a3be683e48a9cd4be4d16275fe5"
+SRCREV = "${AUTOREV}"
```

Comment:
For the commit message: do not use "unfix", but simply state we are moving to AUTOREV here

Reply:
ok